diff -ur a/typecheck/TcPluginM.hs b/typecheck/TcPluginM.hs
--- a/typecheck/TcPluginM.hs	2001-09-09 03:46:40.000000000 +0200
+++ b/typecheck/TcPluginM.hs	2019-11-13 23:54:37.000000000 +0100
@@ -3,7 +3,6 @@
 -- access select functions of the 'TcM', principally those to do with
 -- reading parts of the state.
 module TcPluginM (
-#if defined(GHCI)
         -- * Basic TcPluginM functionality
         TcPluginM,
         tcPluginIO,
@@ -49,10 +48,8 @@
         newEvVar,
         setEvBind,
         getEvBindsTcPluginM
-#endif
     ) where

-#if defined(GHCI)
 import GhcPrelude

 import qualified TcRnMonad as TcM
@@ -190,7 +187,3 @@
 setEvBind ev_bind = do
     tc_evbinds <- getEvBindsTcPluginM
     unsafeTcPluginTcM $ TcM.addTcEvBind tc_evbinds ev_bind
-#else
--- this dummy import is needed as a consequence of NoImplicitPrelude
-import GhcPrelude ()
-#endif
